src/axom/mir/views/dispatch_utilities.hpp

```diff
@@ -45,10 +45,7 @@ constexpr int select_dimensions(Dimensions... dims)
   return encode_dimensions((1 << dims)...);
 }
 
-constexpr bool dimension_selected(int encoded_dims, int dim)
```

Comment:
I'm not a fan of it single-lining a function just because it can.

Reply:
It looks like that's controlled by `AllowShortFunctionsOnASingleLine`.
We don't explicitly specify a value, but the google style that we're based on appears to mark it as `Inline` 

See: 
* https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortfunctionsonasingleline
* https://github.com/kehanXue/google-style-clang-format/blob/master/.clang-format
* https://github.com/LLNL/axom/blob/develop/src/.clang-format